src/SDK/SmartDetectorsPackage/SmartDetectorsPackage.nuproj

```diff
@@ -22,7 +22,7 @@
   <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
   <PropertyGroup Label="Configuration">
     <Id>Microsoft.Azure.Monitoring.SmartDetectors</Id>
-    <Version>0.14.0.0</Version>
+    <Version>0.14.1.0</Version>
```

Comment:
If I remember correctly, the was an additional nuget.
Should we update its version also?

Reply:
no - minor versions can live seperately

---
In reply to: [243895875](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/38#discussion_r243895875) [](ancestors = 243895875)